ui/src/stylesheets/specific/navigation.scss

```diff
@@ -115,12 +108,25 @@
 }
 
 .collapsible {
-  font-weight: var(--weight-bold);
   cursor: pointer;
+
+  &::before{
+    content: url(../images/icons/arrow.svg);
+    margin-right: 0.3em;
+    margin-left: -1.3rem;
+    width: 1rem;
+  }
+
+  &:hover {
+    opacity: .9;
```

Comment:
I fear this might be reducing the opacity doubly for things that are both collapsible headings _and_ links. Note that we don't have an example of this in the UI model but we should because it will happen in the site. Can you add that in the UI model and verify that everything looks fine?

Reply:
Good point, I'll add the example in the UI model and test it 👍

Reply:
Good point, I'll add the example in the UI model and test it 👍